scripts/autolabels.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+ : <<'BASH_MODULE_DOCS'
+
+This file contains the dictionary for converting a modified dir into a github label,
+as well as the script to perform the conversion.
+
+Rules:
+* only lines starting with `|t-` are considered;
+* repeated spaces, commas and pipes (|) are ignored;
+* `Mathlib` is added by default to each `Root folder`.
+
+Exception: `Tactic/Linter` is converted into `Tactic-Linter` internally, to simplify separating it from
```

Comment:
"to easily distinguish it from `Tactic/anything_other_than_linter`"?

Reply:
Also an improvement!